pkg/api/admin_users.go

```diff
@@ -111,8 +111,22 @@ func AdminDeleteUser(c *m.ReqContext) {
 	c.JsonOK("User deleted")
 }
 
+// POST /api/admin/users/:id/disable
+func AdminDisableUser(c *models.ReqContext, cmd dtos.AdminDisableUserCmd) {
+	userID := c.ParamsInt64(":id")
+
+	disableCmd := models.DisableUserCommand{UserId: userID, IsDisabled: cmd.IsDisabled}
```

Comment:
I don't like this approach because it looks like `POST /api/admin/users/:id/disable` without params should disable user, but instead, it's `POST /api/admin/users/:id/disable {"isDisabled": false}` (enable user). Any ideas, @xlson @markelog  @marefr ?

Reply:
@alexanderzobnin to me the payload should be empty and the url/verb describing the intent, i.e. /disable or /enable.

Reply:
I prefer having two endpoints. It reads a lot better vs the other option. 
```
POST /api/admin/users/:id/disable
POST /api/admin/users/:id/enable
```